modules/env/context-always.tf

```diff
@@ -0,0 +1,240 @@
+module "always" {
```

Comment:
did nit get what is this for env, stack, vars sub-modules

please keep things simple, remove all non used codes/modules/files from PRs